scripts/manage/pull_config.sh

```diff
@@ -23,20 +23,20 @@ pushd $TEMP_DIR
 mkdir .hal
 
 # Remove local config so persistent config from Halyard Daemon pod can be copied into place.
-bold "Removing $PARENT_DIR/.hal..."
-rm -rf $PARENT_DIR/.hal
+bold "Removing $HOME/.hal..."
+rm -rf $HOME/.hal
```

Comment:
I tested the setup on a CI server and using `$HOME ` escapes the encapsulation of the workspace when reading and writing `.hal`. What if this script were to use `$PARENT_DIR` only when in CI mode? 

Setup.sh is passed `CI=true` when executed by CI server so I can pass along that value to the pull / push scripts.